zerver/models.py

```diff
@@ -152,6 +164,15 @@ class Realm(ModelReprMixin, models.Model):
     icon_version = models.PositiveSmallIntegerField(default=1)  # type: int
 
     DEFAULT_NOTIFICATION_STREAM_NAME = u'announce'
+    profile_fields = JSONStrField(default=list)  # type: List[Dict[str, Union[int, Text]]]
+
+    def get_profile_field_index(self, field_id):
+        # type: (int) -> int
+        for i, field in enumerate(self.profile_fields):
+            if field['id'] == field_id:
+                return i
```

Comment:
Can you add a docstring or comment explaining how this works?  I'm a bit confused.